instrumentation/github.com/gorilla/mux/otelmux/example/go.mod

```diff
@@ -0,0 +1,11 @@
+module go.opentelemetry.io/opentelemetry-go-contrib/instrumentation/github.com/gorilla/mux/otelmux/example
+
+go 1.14
+
+require (
+	github.com/gorilla/mux v1.8.0
+	go.opentelemetry.io/contrib/instrumentation/github.com/gorilla/mux/otelmux v0.12.0
```

Comment:
In following suit with the other example `go.mod`s this likely need a replace declaration for this as well. When we release breaking changes we will want the example to be updated prior to the release and have this `go.mod` file updatable with the non-existent tag prior to a release.